pagopa/io-app
# chore: [IOPID-1395] Update FeatureInfo font size

## Short description
I've update the version of [io-app-design-system library](https://github.com/pagopa/io-app-design-system/pull/177) with the font adaptation, i've change some text and i've change the rules to show/not show the icon into the opt-in screen and change some texts

<p>


| iPhone 13 mini | iPhone 13 |
| - | - |
 | <video src="https://github.com/pagopa/io-app/assets/83651704/727ae749-63f5-41a8-8080-0c86ef5e833c" /> | <video src="https://github.com/pagopa/io-app/assets/83651704/f3e41573-ccb6-4f0b-9cf7-2e2fb3f1baa0" /> |


</p>



## How to test
Run the application and pay attention to opt-in and error 1002 screens, as you can see from the screen recording.
